packages/grid/x-grid/src/tests/filtering.XGrid.test.tsx

```diff
@@ -1,20 +1,22 @@
-import { expect } from 'chai';
-import * as React from 'react';
-import { useFakeTimers } from 'sinon';
-import { getColumnHeaderCell, getColumnValues } from 'test/utils/helperFn';
-import {
-  createClientRenderStrictMode,
-  // @ts-expect-error need to migrate helpers to TypeScript
-  fireEvent,
-} from 'test/utils';
```

Comment:
Why this diff? Could we minimize them?

On a related note, I believe that so far, the approach is to import the most common modules first, e.g. react, then sinon, then test utils, then the module we test.